include/ppx/metrics.h

```diff
@@ -285,9 +386,24 @@ class Manager final
     // Returns whether a run is active.
     bool HasActiveRun() const;
 
+    // Allocate a MetricID.
+    MetricID AllocateID() { return EnsureAllocateID(kInvalidMetricID); }
+
     // Adds a metric to the current run. A run must be started to add a metric.
     // Failure to add a metric returns kInvalidMetricID.
-    MetricID AddMetric(const MetricMetadata& metadata);
+    // Optionally bind the metric to an existing metricID.
+    MetricID AddMetric(
+        const MetricMetadata& metadata,
+        MetricID              metricID = kInvalidMetricID);
+
+    // Adds a live metric.
+    // Optionally bind the metric to an existing metricID.
+    MetricID AddLiveMetric(
+        double   halfLife = LiveMetric::kDefaultHalfLife,
+        MetricID metricID = kInvalidMetricID);
+
+    // Adds a live metric to current run.
+    void BindMetric(MetricID liveMetricID, const MetricMetadata& metadata);
```

Comment:
Is this defined somewhere?

Reply:
My bad, I should have be more careful removing temporary code.